tests/pilot_benchmark/snakemake/configs/samples.csv

```diff
@@ -0,0 +1,7 @@
+group,replicate,fastq_1,fastq_2,strandedness,sample,bam
+siControl,1,data/fastqs/SRR11918577.fastq.gz,,unstranded,siControl_1,../nf_core_qc/results/star/siControl_R1.markdup.sorted.bam
```

Comment:
Those should point to test files that are _included_ in the repo, *NOT* to paths on your local machine. In that way, others can run the pilot benchmark to make sure they have properly set up their system to start writing execution workflows

Reply:
see comment above.